packages/shared/src/components/search/SearchResults/SearchResultsTags.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { TagLink } from '../../TagLinks';
+import { ListItemPlaceholder } from '../../widgets/ListItemPlaceholder';
+import { SearchSuggestion } from '../../../graphql/search';
+
+interface SearchResultsTagsProps {
+  items: string[];
+  isLoading: boolean;
+  onTagClick: (suggestion: SearchSuggestion) => void;
+}
+
+export const SearchResultsTags = (
+  props: SearchResultsTagsProps,
+): ReactElement => {
+  const { items = [], isLoading, onTagClick } = props;
+
+  if (!isLoading && !items.length) {
+    return null;
+  }
+
+  return (
+    <WidgetCard heading="Related tags">
+      {!!items?.length && (
+        <div className="flex flex-wrap gap-3" role="list">
+          {items.map((tag) => (
+            <TagLink
+              key={tag}
+              tag={tag}
```

Comment:
I think you are using the wrong value here.
It renders as the name, but should be the tag value to be consistent.

![Screenshot 2024-08-01 at 09 57 29](https://github.com/user-attachments/assets/7ffc123c-6873-41e3-bb3c-a9db98e9c21d)

Reply:
Updated ✅